.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -0,0 +1,97 @@
+---
+name: Server migration
+about: Checklist to migrate to an upgraded server
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Checklist based on general guide https://github.com/openfoodfoundation/ofn-install/wiki/Migrating-a-Production-Server
+
+## 1. Setting up the new server
+- [ ] Check old server config for any additional services to be aware of (eg `/etc/nginx/sites-available`). Document any necessary steps for migration.
+- [ ] Hosting: provision new server with Ubuntu 20
+- [ ] DNS: add temporary domain (eg `prod2.openfoodnetwork.org`)
+
+### config
+- [ ] Add temporary name to `inventory/hosts`
+- [ ] Review `host_vars/x/config.yml`, clean up if needed
+  - Make a copy for the temp hostname, add temp domain to bottom of `certbot_domains`
+- [ ] Review `ofn-secrets:x_prod/secrets.yml`, clean up if needed
+   - Change to shared bugsnag projects
+   - Don't bother making a copy of this one
+
+### setup
+Ensure you have the correct secrets.
+`ansible-playbook -l x_prod2 -e "@../ofn-secrets/x_prod/secrets.yml playbooks/`
+- [ ] `letsencrypt_proxy.yml`
+- [ ] `setup.yml`
+- [ ] `provision.yml`
+- [ ] `deploy.yml`
+- [ ] `db_integrations` (Permit DB access for n8n, Metabase)
+- [ ] Ensure sidekiq is disabled, to avoid creating subscription orders: `systemctl disable sidekiq`
```

Comment:
Disable and stop in this case.

But I wonder if we should avoid copying production data before the switchover. :thinking: 

I guess it's part of the switchover and if problems are identified then we can fix it, go back and start again.

Reply:
Hmm I guess it's a question of if you want to separate out the testing from the switchover. Ideally, we wouldn't need to: everything just works and we setup, copy db and proceed to migrate and it's all good.

But we need to test and validate that things are working. And often, there will be something not working.
I'd rather identify that before co-ordinating and communicating a downtime, and putting the old site into maintenance mode. 
Maybe for some smaller instances, we don't need that added layer of testing, and can accept more downtime while we frantically fix things quickly in-place. That might be a way more efficient use of time. (we ended up doing that for CA anyway!)
But I don't think that would be appropriate in all cases. Particularly for CA, I think it would have been too much to deal with all at once and we may have had to abort.